api/lang/en/placement_type.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'place_tentative' => 'Tentatively placed (Verbal or email confirmation)',
+    'placed_casual' => 'Placed casual',
+    'placed_term' => 'Placed term',
+    'placed_indeterminate' => 'Placed indeterminate',
```

Comment:
Need to fix a typo for `place_tentative` to `placed_tentative`

Reply:
Same with the placement.fr file!

Reply:
Fixed in [fix typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10701/commits/75256dc3df5c2b5dd95ae16fe44dba5c41f09b30)